tools/harvest_source_import/tests/test_import_sources.py

```diff
@@ -0,0 +1,41 @@
+import pytest
```

Comment:
I think this file should be called _import_test_sources.py_

Reply:
If I rename it pytest you won't see the file